MatrixSDKExtensions/VoIP/Jingle/MXJingleCallStackCall.m

```diff
@@ -27,6 +27,8 @@
 #import "MXJingleCameraCaptureController.h"
 #import <WebRTC/WebRTC.h>
 
+NSString *const kMXJingleCallWebRTCMainStreamID = @"ARDAMS";
```

Comment:
I'm going to suggest 'userMedia' for now: anything would be better than 'ARDAMS'

Reply:
👍